go/integTest/shared_commands_test.go

```diff
@@ -3544,6 +3544,33 @@ func (suite *GlideTestSuite) TestPExpireTime() {
 	})
 }
 
+func (suite *GlideTestSuite) Test_ZCard() {
+	suite.runWithDefaultClients(func(client api.BaseClient) {
+		key := "{key}" + uuid.NewString()
+		membersScores := map[string]float64{
+			"one":   1.0,
+			"two":   2.0,
+			"three": 3.0,
+		}
+
+		res1, err := client.ZAdd(key, membersScores)
+		assert.Nil(suite.T(), err)
```

Comment:
```suggestion
		assert.Nil(t, err)
```
Similarly, for all assert statements below.